swap/src/asb/config.rs

```diff
@@ -139,6 +150,8 @@ pub fn query_user_for_initial_testnet_config() -> Result<Config> {
         bitcoin: Bitcoin { electrum_rpc_url },
         monero: Monero {
             wallet_rpc_url: monero_wallet_rpc_url,
+            wallet_name: monero_wallet_name,
+            wallet_password: monero_wallet_password,
```

Comment:
This stores the password in plaintext on the disk ...

Can we change this to an env variable? E.g. if the env variable is set, we use it as the password, otherwise we default to empty.

Reply:
Yes, it does. We also store the seed there which is basically the password for the Bitcoin wallet.
How is this a worse security assumption?